editor/plugins/script_editor_plugin.cpp

```diff
@@ -2063,6 +2064,11 @@ void ScriptEditor::_update_script_names() {
 				case SORT_BY_PATH: {
 					sd.sort_key = path;
 				} break;
+				case SORT_BY_MODIFIED: {
```

Comment:
```suggestion
				case SORT_BY_MODIFIED_TIME: {
```
You forgot to update this one

Reply:
Ooops, missed one. Thanks.